### DISCUSSION TITLE
Buttons to control animation in AR view

### DISCUSSION THREAD

**Discussion by DanieleMagri** [google/model-viewer#4390] [Q&A]
Is it possible to display two buttons "play" and "stop" inside the AR view (using WebXR obviously) to control the animation of the model inside the AR scene?

Nothing fancy, literally two html/css buttons with the AR scene behind them.

**Comment by danilokrulj** [comment]
Did you try? If my memory isn't wrong i think its possible, but it would only work using WebXR.

**Reply by DanieleMagri** [reply]
Not yet, I'll try eventually, I should put the  buttons in the 3D viewport of model-viewer if I'm not mistaken, right?

**Reply by danilokrulj** [reply]
I just tested it inside the model-viewer div with position absolute and its visible in the AR scene. I didn't test the animation part but I'm pretty sure it should work.

**Reply by DanieleMagri** [reply]
Very nice, thank you :)
